cit3.0-web/src/App.js

```diff
@@ -11,7 +11,7 @@ import SiteInfomation from "./components/Page/SiteInformation/SiteInformation";
 import PropertyDetails1 from "./components/Page/PropertyDetails1/PropertyDetails1";
 import PropertyDetails2 from "./components/Page/PropertyDetails2/PropertyDetails2";
 import ReviewOpportunity from "./components/Page/ReviewOpportunity/ReviewOpportunity";
-import Flyout from "./components/Flyout/Flyout";
+// import Flyout from "./components/Flyout/Flyout";
```

Comment:
This is okay to remove, there was a new component that wraps this now. App no longer controls it. :)